tests/test_groups.py

```diff
@@ -71,16 +72,16 @@ def test_get_group_error_not_found():
 def test_get_group_success(mocker):
     responses.add(
         responses.GET,
-        f"{URL}/api/security/groups/{NEW_GROUP.name}",
-        json=NEW_GROUP.dict(),
+        f"{URL}/api/security/groups/{NEW_GROUP.name}?includeUsers=True",
+        json=GROUP_WITH_USERS.dict(),
         status=200,
     )
 
     artifactory_group = ArtifactoryGroup(AuthModel(url=URL, auth=AUTH))
     mocker.spy(artifactory_group, "get")
-    artifactory_group.get(NEW_GROUP.name)
+    group = artifactory_group.get(NEW_GROUP.name)
 
-    artifactory_group.get.assert_called_with(NEW_GROUP.name)
+    assert group.dict() == GROUP_WITH_USERS.dict()

```

Comment:
If the user has an old version of artifactory, I'm not sure that this is the result we'll get.